client/vite.config.ts

```diff
@@ -1,7 +1,35 @@
 import { defineConfig } from 'vite';
 import react from '@vitejs/plugin-react';
-
+import { getThemeVariables } from 'antd/dist/theme';
```

Comment:
This import is not needed, please delete it.

Reply:
Done!